tests/unit/test_transform_helpers.py

```diff
@@ -30,8 +33,9 @@ def test_month_dim_rows(transform):
 
 
 def test_date_dim_row(transform, date_dim_rows):
-    rows = transform.date_dim_row()
-    rows == [date_dim_rows]
+    with freeze_time("2023-08-01"):
```

Comment:
nice, used to decorator but it has a context manager as well :)

Reply:
I first tried with the decorator but didn't work as expected so I tried with the context manager and worked. My first time working freeze_time so very possible I didn't setup decorator correctly. What made it a little different from documentation scenarios is date is being set within the `date_dim_row` method of the test function ... not explicitly within the test function like `test_date = datetime(...)` - this is what made me try out the context manager after trying the decorator